i18n/en/docusaurus-plugin-content-docs/current/03-消费者/13pull.md

```diff
@@ -86,9 +86,10 @@ public class LitePullConsumerSubscribe {
 }
 ```
 
-首先还是初始化`DefaultLitePullConsumer`并设置`ConsumerGroupName`，调用subscribe方法订阅topic并启动。与Push Consumer不同的是，`LitePullConsumer`拉取消息调用的是轮询poll接口，如果能拉取到消息则返回对应的消息列表，否则返回null。通过`setPullBatchSize`可以设置每一次拉取的最大消息数量，此外如果不额外设置，`LitePullConsumer`默认是自动提交位点。在subscribe模式下，同一个消费组下的多个`LitePullConsumer`会负载均衡消费，与PushConsumer一致。
+First of all, initialize `DefaultLitePullConsumer` and set `ConsumerGroupName`, call subscribe method to subscribe to the topic and start it. Unlike Push Consumer, `LitePullConsumer` pulls messages by polling the poll interface and returns the corresponding message list if it can pull the message, otherwise it returns null. The maximum number of messages per pull can be set with `setPullBatchSize`, and `LitePullConsumer` automatically commits the bit by default if not set additionally. In subscribe mode, multiple ` LitePullConsumer` under the same consumer group are load-balanced for consumption, consistent with PushConsumer.
```

Comment:
> First of all, initialize `DefaultLitePullConsumer` and set `ConsumerGroupName`, call subscribe method to subscribe to the topic and start it. Unlike Push Consumer, `LitePullConsumer` pulls messages by polling the poll interface and returns the corresponding message list if it can pull the message, otherwise it returns null. The maximum number of messages per pull can be set with `setPullBatchSize`, and `LitePullConsumer` automatically commits the bit by default if not set additionally. In subscribe mode, multiple ` LitePullConsumer` under the same consumer group are load-balanced for consumption, consistent with PushConsumer.

to:

> First of all, initialize `DefaultLitePullConsumer` and set `ConsumerGroupName`. Call the subscribe method afterward to subscribe to a topic and start it. Unlike the Push Consumer, `LitePullConsumer` pulls messages by the poll interface and returns the corresponding message list if it can pull the message, otherwise, it returns null. The maximum number of messages per pull can be set with `setPullBatchSize`, and the `LitePullConsumer` will automatically commits the offset by default. In the Subscribe mode, multiple ` LitePullConsumer` under the same consumer group are load-balanced for consumption, consistent with the PushConsumer.